r2i/onnxrt/parameters.h

```diff
@@ -40,89 +41,21 @@ class Parameters: public IParameters {
   RuntimeError Set (const std::string &in_parameter, int in_value) override;
   RuntimeError Set (const std::string &in_parameter, double in_value) override;
   RuntimeError List (std::vector<ParameterMeta> &metas) override;
+  /* To set/get Parameters of the internal Engine instance */
+  RuntimeError SetLogLevel (int &value);
```

Comment:
Setters shouldn't  receive by reference (except for the string, for which I recommend `const std::string`

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/f60ee705f623ed661e84f61aa9f3b3a541adcc09)